CommentingPractice/services/encode-decode.js

```diff
@@ -51,3 +51,4 @@ function decode(input){
     return (decodedString);
 
 }
+export { dict, encode, getKeyByValue,decode}
```

Comment:
I think we only need to export encode and decode because those are the only two methods that the consumer of your module needs to make use of. Dict and getKeyByValue are details that the consumer doesn't need to have access to.